packages/jaeger-ui/src/components/TracePage/TracePageHeader/AltViewOptions.test.js

```diff
@@ -68,6 +68,16 @@ describe('AltViewOptions', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
+  it('json links are not displayed because forbidnewPagePage is true', () => {
```

Comment:
This seems like unrelated coupling, JSON can still be shown, just open in the same window (and require Back button to get back into the UI)